pkg/virtctl/vmexport/vmexport.go

```diff
@@ -100,16 +100,22 @@ var (
 	insecure     bool
 	keepvme      bool
 	volumeName   string
-
-	// vmexport info
-	vmexportName string
-	funcName     string
-	exportSource k8sv1.TypedLocalObjectReference
 )
 
-type exportFunc func(client kubecli.KubevirtClient, namespace string) error
+type exportFunc func(client kubecli.KubevirtClient, vmeInfo *VMExportInfo) error
 
-type HTTPClientCreator func(*http.Transport) *http.Client
+type HTTPClientCreator func(*http.Transport, bool) *http.Client
+
+type VMExportInfo struct {
+	shouldCreate bool
```

Comment:
need to do all this variables also starting in CAPS so they will be able to change in other package